tests/unit_tests/c_wrappers/res/enkf/test_res_config.py

```diff
@@ -624,3 +624,34 @@ def run(self, *args):
 
     assert ert_config.workflow_jobs["plugin"].isPlugin()
     assert not ert_config.workflow_jobs["script"].isPlugin()
+
+
+def test_data_file_with_non_utf_8_character_gives_error_message(tmpdir):
+    with tmpdir.as_cwd():
+        data_file = "data_file.DATA"
+        with open("config.ert", mode="w", encoding="utf-8") as fh:
+            fh.write(
+                """NUM_REALIZATIONS 1
```

Comment:
Quick question, any good reason why not dedent this? Also why different indent?

Reply:
I think that is not needed as long as you use `"""`